strings_lab.js

```diff
@@ -0,0 +1,227 @@
+// # String Exercises
+//
+// 1. console.log the first character of a string.
```

Comment:
Good demonstration of your understanding of string methods and properties to solve problems. I like how you tried different JS syntaxes like ternary, function, and regex. Keep up the good work!